modules/get-started/pages/rpk-first-steps.adoc

```diff
@@ -0,0 +1,240 @@
+See xref:reference:rpk/rpk-topic/rpk-topic-produce.adoc[].
+
+== Get a description of a topic
+
+Check the topic’s configuration and status to ensure that it’s ready for use:
+
+[source,bash]
+----
+rpk topic describe tutorial
+----
+
+Output:
+[source,bash]
+----
+SUMMARY
+=======
+NAME        tutorial
+PARTITIONS  1
+REPLICAS    1
+
+CONFIGS
+=======
+KEY                                   VALUE       SOURCE
+cleanup.policy                        delete      DEFAULT_CONFIG
+compression.type                      producer    DEFAULT_CONFIG
+delete.retention.ms                   -1          DEFAULT_CONFIG
+flush.bytes                           262144      DEFAULT_CONFIG
+flush.ms                              100         DEFAULT_CONFIG
+initial.retention.local.target.bytes  -1          DEFAULT_CONFIG
+initial.retention.local.target.ms     -1          DEFAULT_CONFIG
+max.message.bytes                     1048576     DEFAULT_CONFIG
+message.timestamp.type                CreateTime  DEFAULT_CONFIG
+redpanda.iceberg.delete               true        DEFAULT_CONFIG
+redpanda.iceberg.mode                 disabled    DEFAULT_CONFIG
+redpanda.leaders.preference           none        DEFAULT_CONFIG
+redpanda.remote.delete                true        DEFAULT_CONFIG
+redpanda.remote.read                  false       DEFAULT_CONFIG
+redpanda.remote.write                 false       DEFAULT_CONFIG
+retention.bytes                       -1          DEFAULT_CONFIG
+retention.local.target.bytes          -1          DEFAULT_CONFIG
+retention.local.target.ms             86400000    DEFAULT_CONFIG
+retention.ms                          604800000   DEFAULT_CONFIG
+segment.bytes                         134217728   DEFAULT_CONFIG
+segment.ms                            1209600000  DEFAULT_CONFIG
+write.caching                         true        DEFAULT_CONFIG
+----
+
+See xref:reference:rpk/rpk-topic/rpk-topic-describe.adoc[].
+
+== Consume records from a topic
+
+Consume records from the topic:
+
+[source,bash]
+----
+rpk topic consume tutorial
+----
+
+Output:
+[source,json]
+----
+{ "topic": "tutorial", "value": "hello", "timestamp": 1678807229837, "partition": 0, "offset": 0 }
+{ "topic": "tutorial", "value": "world", "timestamp": 1678807232413, "partition": 0, "offset": 1 }
+----
+
+Consume from an offset, where `2` is not inclusive: 
+[source,bash]
+----
+rpk topic consume tutorial --offset 0:2
+----
+Output:
+[source,json]
+----
+{ "topic": "tutorial", "value": "hello", "timestamp": 1678807229837, "partition": 0, "offset": 0 }
+{ "topic": "tutorial", "value": "world", "timestamp": 1678807232413, "partition": 0, "offset": 1 }
+----
+
+See xref:reference:rpk/rpk-topic/rpk-topic-consume.adoc[].
+
+== Create a consumer group and consume topics
+
+Organize consumers into groups to share workloads and balance consumption:
+
+[source,bash]
+----
+rpk topic consume tutorial --group tutorial-group
+----
+
+NOTE: The consumer group is created when you consume from the topic.
```

Comment:
```suggestion
NOTE: The consumer group is created when you start consuming from the topic.
```

Probably better, wdyt? as `when you consume` may imply that only is created when you consume records, but the consumer might be idle waiting for new messages and the consumer group still will be created